src/ape/api/providers.py

```diff
@@ -588,6 +588,18 @@ def revert(self, snapshot_id: SnapshotID):
             snapshot_id (str): The snapshot ID.
         """
 
+    @abstractmethod
+    def fast_forward_time(self, new_timestamp: int) -> int:
```

Comment:
Seems better to just do `set_timestamp` and let the provider plugin specify what range of timestamps it supports